src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -448,9 +525,19 @@ async function InitializeFirebaseAdminStubs(frb: FunctionsRuntimeBundle): Promis
         return adminModuleTarget.initializeApp(opts, appName);
       }
 
-      new EmulatorLog("SYSTEM", "default-admin-app-used", "").log();
+      const config = JSON.parse(process.env.FIREBASE_CONFIG || "{}");
+      new EmulatorLog("SYSTEM", `default-admin-app-used, config=${config}`, "").log();
+
+      // TODO: Is there any possible harm in this?
```

Comment:
Anyone wanna tell me how this could go wrong?  Seems to be required to make RTDB work and not affect Firestore.